Screens/cart.css

```diff
@@ -0,0 +1,112 @@
+.cart-container{
+    display: flex;
+    margin: auto;
+    width: 80%;
+    overflow-x: auto ;
+}
+
+.cart-container .cart-table{
+    width: 100%;
+    border-collapse: collapse;
+    table-layout: fixed;
+    white-space: nowrap;
+}
+
+.cart-container .cart-table thead{
+    border: var(--border-regular);
+    border-right: none;
+    border-left: none;
+}
+
+.cart-container .cart-table thead td{
+    font-weight: bold;
+    text-transform: uppercase;
+    font-size: 1.4rem;
+    padding: 1.5rem 0;
+}
+
+.cart-container .cart-table tbody tr td{
+    padding-top: 1rem;
+}
+
+.cart-container .cart-table tbody td{
+    font-size: 1.2rem;
+}
+
+.cart-container .cart-table img{
+    width: 6rem;
+}
+
+.cart-container .cart-table td:nth-child(1){
+    width: 10rem;
+    text-align: center;
+}
+
+.cart-container .cart-table td:nth-child(2),
+.cart-container .cart-table td:nth-child(4),
+.cart-container .cart-table td:nth-child(5),
+.cart-container .cart-table td:nth-child(6){
+    width: 14rem;
+    text-align: center;
```

Comment:
as you are using text-align repeatedly, you can add it your utility classes(just a suggestion)

Reply:
ok